pkgs/games/solicurses/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, ncurses
+}:
+
+stdenv.mkDerivation rec {
+  pname = "solicurses";
+  version = "unstable-2020-02-13";
+
+  src = fetchFromGitHub {
+    owner = "KaylaPP";
+    repo = "SoliCurses";
+    rev = "dc89ca00fc1711dc449d0a594a4727af22fc35a0";
+    sha256 = "sha256-zWYXpvEnViT/8gsdMU9Ymi4Hw+nwkG6FT/3h5sNMCE4=";
+  };
+
+  buildInputs = [
+    ncurses
+  ];
+
+ # makefile = "build/Makefile";
+ preBuild = ''
+   cd build
+ '';
+  installPhase = ''
+    mkdir -p $out/bin
+    ls -R
+    install SoliCurses.out $out/bin/solicurses
+  '';
```

Comment:
```suggestion

installPhase = ''
  install SoliCurses.out $out/bin/solicurses
'';
```

Reply:
admited, the `ls -R` was for debugging and isn't necessary, but without the `mkdir` the `install` fails with "no such file or directory"

Reply:
Sorry, I forgot the flag `-D`.

The command is `install SoliCurses.out -D $out/bin/solicurses`
 `